src/git.sh

```
@@ -1,42 +1,60 @@
 #     echo "pr_message=${pr_message}"
 #     echo "pr_message<<EOF" >> $GITHUB_OUTPUT
 #     echo "${pr_message}" >> $GITHUB_OUTPUT
 #     echo "EOF" >> $GITHUB_OUTPUT
 #   fi
-
 #   echo "${pr_message}"
 # }
 
 git_get_last_created_tag() {
+  # Parse arguments using the new parse_arguments function
+  local args_json=$(parse_arguments "$@")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)